pluginconfig/commit.go

```diff
@@ -22,6 +22,9 @@ type CommitPluginConfig struct {
 	// NewMsgScanBatchSize is the number of max new messages to scan, typically set to 256.
 	NewMsgScanBatchSize int `json:"newMsgScanBatchSize"`
 
+	// The maximum number of times to check if the previous report has been transmitted
+	MaxReportTransmissionCheckAttempts uint
```

Comment:
Is this something we want all plugin instances in the DON to have in common? If so, it should go in the OffchainConfig struct below

Reply:
What's the difference between `CommitPluginConfig` and `CommitOffchainConfig`?